### ISSUE TITLE
automatically download geth binary

### ISSUE THREAD

**Issue by mastilver** [ethereumjs/geth.js#2]
Hi

I just created: https://github.com/mastilver/geth-bin

If this module depends on `geth-bin` then `geth` will be installed automatically and directly available
It's a weird feature, so I wanted to check with you what you think about it before sending you a PR

**Comment by tinybike** [comment]
Cool!  Yes, I think that automatically downloading geth could be useful.  It would be good if it could grab the latest version when it downloads -- maybe (getting a bit fancy here,  but...) it could have a `geth.checkForUpdates()` function that downloads a new build if one exists.

I think for this to be useful, though, one thing it would also need to do is handle the initial account creation.  I suspect that people downloading geth generally haven't set up their data directory yet!

**Comment by mastilver** [comment]
> one thing it would also need to do is handle the initial account creation
Couldn't we use https://github.com/ethereum/web3.js to handle account creation

**Comment by kumavis** [comment]
@mastilver we're interested in a PR if you're still maintaining the project - thanks!

**Comment by tinybike** [comment]
Yes, I don't have time to write this at the moment, but would be more than happy to merge a PR :)
